selfdrive/car/hyundai/values.py

```diff
@@ -113,6 +113,7 @@ class CAR(StrEnum):
   KIA_NIRO_EV = "KIA NIRO EV 2020"
   KIA_NIRO_EV_2ND_GEN = "KIA NIRO EV 2ND GEN"
   KIA_NIRO_PHEV = "KIA NIRO HYBRID 2019"
+  KIA_NIRO_PHEV_2021 = "KIA NIRO PLUG-IN HYBRID 2021"
```

Comment:
Any reason this can't be KIA_NIRO_PHEV_2022? We'd prefer to combine platforms as it improves fingerprinting, and then fall back to detecting features.

Reply:
The 2021 and 22 use different harnesses and I didn't see a clean way to make them the same car.   As my changes propose, I would at least like the 2021 and 22 broke out as a car separate from earlier years of the PHEV, because there's at least one feature like minimum cruise control speed (0 mph for the above vs. 32 for 2020 and earlier), and the car mass is much lower than the average that includes earlier EV models.

Reply:
BTW now I don't know why the new provided test route is failing.   It's uploaded, as I see it in Connect, and I marked it public and preserved.

Reply:
BTW I don't know why the test route check is failing.  It's preserved and public.  In useradmin the route name has a forward slash in it and I tried changing it (the typo fix) to vertical bar - still failed.

Reply:
You can just made the car info value a list.

Reply:
OK, I think I understand and will go back and do that.  Any idea about the test route fail, or is that not important?